hack/aks

```diff
@@ -14,13 +14,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-AZURE_CLI="az"
 TERRAFORM="terraform"
 KUBECTL="kubectl"
 RUN_PATH="$HOME/tmp/terraform-aks"
 CONFIG_PATH=$(dirname $(readlink -f "$0"))/terraform/aks
-CMD_ARRAY=($AZURE_CLI $TERRAFORM $KUBECTL pv bzip2 jq)
-ENV_ARRAY=(TF_VAR_aks_client_id TF_VAR_aks_client_secret)
+CMD_ARRAY=($TERRAFORM $KUBECTL pv bzip2 jq)
+ENV_ARRAY=(TF_VAR_azure_client_id TF_VAR_azure_client_secret TF_VAR_azure_client_subscription_id TF_VAR_azure_client_tenant_id)
```

Comment:
At line 84 to 87, we are installing check in version of antrea and cert manager. As we dont require any local changes on that yaml, we can consider pulling the released 1.8 version directly. Declare version variable at the top and apply them here. Or create a new file of antrea and cert manager version, and source it here. In this same, we can update the version at one place, and all eks/aks and kind script can refer to it.

Reply:
As discussed offline, we need to use checked in version of yamls since local changes are needed.